package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "open-map-kit-server",
-  "version": "0.7.0",
+  "version": "0.8.0",
```

Comment:
This probably shouldn't be bumped in a pull request, preferring to tag a release using `npm version ...` off of the primary branch once it's ready.